tests/blackbox/stratis_cli_cert.py

```diff
@@ -26,13 +26,14 @@
 DISKS = []
 
 
-def make_test_pool():
+def make_test_pool(pool_disks):
     """
     Create a test pool that will later get destroyed
+    :param list pool_disks: List of disks with which the pool will be created
     :return: Name of the created pool
     """
     pool_name = p_n()
-    assert (exec_test_command([STRATIS_CLI, "pool", "create", pool_name, DISKS[0]]))[
+    assert (exec_test_command([STRATIS_CLI, "pool", "create", pool_name, pool_disks]))[
```

Comment:
The place to look for the bug is just here. You documented ```pool_disks``` to be a list of str, essentially, but you use it as if it a str. It is correct to call ```pool_disks``` a list of str. So the error lies in the fact that the code within the method treats ```pool_disks``` as if it were a str.